parser/README.md

```diff
@@ -0,0 +1,161 @@
+# Python interface to Stanford Core NLP tools v3.4.1
+
+This is a Python wrapper for Stanford University's NLP group's Java-based [CoreNLP tools](http://nlp.stanford.edu/software/corenlp.shtml).  It can either be imported as a module or run as a JSON-RPC server. Because it uses many large trained models (requiring 3GB RAM on 64-bit machines and usually a few minutes loading time), most applications will probably want to run it as a server.
```

Comment:
I think that you have included the CoreNLP python library python files - you don't want to do that. You probably want a requirements.txt file, and to specify dependencies there via pip. If that doesn't work or if the library isn't available on pip, then I would suggest using Python Buildout to specify how to download and install the dep.